deepmd/infer/deep_eval.py

```
@@ -93,6 +93,19 @@ def __init__(
     ) -> None:
         pass
 
+    def close(self) -> None:
+        """Release resources held by the backend.
+
+        The base implementation does nothing. Backends that hold persistent
+        resources (such as a TensorFlow session) should override it.
+        """
```

Ruff (0.15.20)

[warning] `DeepEvalBackend.close` is an empty method in an abstract base class, but has no abstract decorator

(B027)